tsdb/index/inmem/meta.go

```diff
@@ -19,43 +19,43 @@ import (
 // contains in memory structures for indexing tags. Exported functions are
 // goroutine safe while un-exported functions assume the caller will use the
 // appropriate locks.
-type Measurement struct {
-	database string
-	Name     string `json:"name,omitempty"`
-	name     []byte // cached version as []byte
+type measurement struct {
+	Database  string
+	Name      string `json:"name,omitempty"`
+	NameBytes []byte // cached version as []byte
```

Comment:
Do we need both a string and a byte version of name?  Wonder if we can just have `Name []byte`.

Reply:
👍 I'll take a look and adjust if `Name` is no longer needed

Reply:
`Name` is still used in a number of places, however I pushed an additional commit that replaces `[]byte(Measurement.Name)` with `Measurement.NameBytes` to save allocations

Reply:
Cool.  Can remove the other places some other time.